pkg/splunk/common/urls.go

```diff
@@ -10,94 +10,33 @@ const (
 	//ClusterManager = "cluster-master"
 	ClusterManager = "cluster-master"
 
-	//URIClusterManagerClusterConfig = "/services/cluster/config"
-	URIClusterManagerClusterConfig = "/services/cluster/config"
-
-	//URICLusterManagerServices = "/services/cluster/master"
-	URICLusterManagerServices = "/services/cluster/master"
-
-	//URIClusterManagerGetInfo = "/services/cluster/master/info"
-	URIClusterManagerGetInfo = URICLusterManagerServices + "/info"
-
-	//URIClusterManagerGetPeers = "/services/cluster/master/peers"
-	URIClusterManagerGetPeers = URICLusterManagerServices + "/peers"
-
-	//URIClusterManagerRemovePeers = "/services/cluster/master/control/control/remove_peers"
-	URIClusterManagerRemovePeers = URICLusterManagerServices + "/control/control/remove_peers"
-
-	//URIClusterManagerApplyBundle = "/services/cluster/master/control/default/apply"
-	URIClusterManagerApplyBundle = URICLusterManagerServices + "/control/default/apply"
-
-	//URIClusterManagerGetHealth = "/services/cluster/master/health"
-	URIClusterManagerGetHealth = URICLusterManagerServices + "/health"
-
-	//URIClusterManagerGetSites = "/services/cluster/master/sites"
-	URIClusterManagerGetSites = URICLusterManagerServices + "/sites"
-
 	//URIClusterManagerGetSearchHeads = "/services/cluster/master/searchheads"
```

Comment:
Just double-checking to see if these were indented to be removed.

Reply:
Yes. As part of the effort we went through the list of all APIs and updated them based on the 9.0 release, which already supports `manager` in many scenarios. For example `"/services/cluster/manager"` now is widely supported.